### DISCUSSION TITLE
Support Android 6

### DISCUSSION THREAD

**Discussion by shamefulCake1** [plateaukao/einkbro#394] [Ideas]
Hello,

Is it possible (hard) to add support for Android 6?

I am using Onyx Boox Max 2, and it is an otherwise wonderful eink device, but Android on it is rather old.

**Comment by plateaukao** [comment]
It takes too much time to maintain ancient Android version devices.
Yes, it's possible; if you wanna do so. 
As for me, since I don't have any Android 6 devices, I may not spend time on doing so.
